app/services/user.js

```diff
@@ -1,6 +1,7 @@
 const { user: User } = require('../models'),
   { databaseError } = require('../errors'),
-  { DEFAULT_PAGE_LIMIT } = require('../constants');
+  { DEFAULT_PAGE_LIMIT } = require('../constants'),
+  { userAlbum: UserAlbum } = require('../models');
 
 exports.createUser = ({ email, ...otherUserData }, next) =>
```

Comment:
theres no need to call next in the services, you can just throw and the catch in the controller can get it and call next then, without the service needing to know what next is.